R/derive_param_exposure.R

```diff
@@ -6,15 +6,42 @@
 #' @param dataset
 #'   `r roxygen_param_dataset(expected_vars = c("by_vars", "analysis_var"))`
 #'   `PARAMCD` is expected as well,
-#'   + Either `ASTDTM` and `AENDTM` or `ASTDT` and `AENDT` are also expected.
 #'
-#' @param filter Filter condition
+#' @param dataset_add Additional dataset
 #'
-#'   The specified condition is applied to the input dataset before deriving the
-#'   new parameter, i.e., only observations fulfilling the condition are taken
-#'   into account.
+#'   The variables specified for `by_vars`, `analysis_var`,
+#'    + alongside either `ASTDTM` and `AENDTM` or `ASTDT` and `AENDT` are also expected.
```

Comment:
I think the "+" should be removed (the rendered versions looks strange).

And `PARAMCD` should be added.